analytics-webapps/src/main/webapp/vue-app/generic-portlet/settings/form/FieldSelection.vue

```diff
@@ -104,6 +104,11 @@ export default {
       return this.fieldNames.filter(field => (!this.aggregation || field.aggregation) && (!this.numeric || field.numeric));
     },
   },
+  watch : {
+    fields(){
+      this.fields = this.fields.sort((a, b) => a.label.toLowerCase().localeCompare(b.label.toLowerCase()));
```

Comment:
you change the same field in his `watch` expression. This is likely an infinite loop isn't it ?